lxml-stubs/etree.pyi

```diff
@@ -692,3 +692,64 @@ class TreeBuilder:
     def start(self, tag: _AnyStr, attrib: Dict[_AnyStr, _AnyStr]) -> None: ...
 
 def iselement(element: Any) -> TypeGuard[_Element]: ...
+
+_ParseEventType: TypeAlias = Literal[
+    "start", "end", "start-ns", "end-ns", "comment", "pi"
+]
+_ParseEvent: TypeAlias = Union[
+    tuple[Literal["start"], _Element],
+    tuple[Literal["end"], _Element],
+    tuple[Literal["start-ns"], Tuple[_AnyStr, _AnyStr]],
+    tuple[Literal["end-ns"], None],
+    tuple[Literal["comment"], _Comment],
+    tuple[Literal["pi"], _ProcessingInstruction],
+]
+
+class XMLPullParser(XMLParser):
+    def __init__(
+        self,
+        events: Optional[tuple[_ParseEventType, ...]] = ...,
```

Comment:
This doesn't have to be a tuple (same below). You can pass any iterable. A list would actually be more natural, in fact.

Reply:
You are entirely right, corrected.